packages/react-aria-components/src/Calendar.tsx

```diff
@@ -77,6 +77,8 @@ export const CalendarContext = createContext<ContextValue<CalendarProps<any>, HT
 export const RangeCalendarContext = createContext<ContextValue<RangeCalendarProps<any>, HTMLDivElement>>({});
 export const CalendarStateContext = createContext<CalendarState | null>(null);
 export const RangeCalendarStateContext = createContext<RangeCalendarState | null>(null);
+const InternalCalendarContext = createContext<CalendarProps<any> | null>(null);
```

Comment:
should we just send this on the normal state context?

Reply:
Are you suggesting adding it to `CalendarState`, or adding the value to `CalendarStateContext`?